tests/test_code_hasher.py

```diff
@@ -192,6 +192,30 @@ class ClassWithClassMethod2:
         def v(cls):
             return 20
 
+    class ClassWithProperty:
```

Comment:
Should these two classes get added to `values`?

Reply:
Yes 🤦🏽